src/Sylius/Behat/Service/Accessor/NotificationAccessor.php

```diff
@@ -108,4 +108,14 @@ private function assertElementExistsOnPage(NodeElement $element)
             throw new ElementNotFoundException(sprintf('%s element is not present on the page', self::NOTIFICATION_ELEMENT_CSS));
         }
     }
+
+    /**
+     * @param string $resourceName
+     *
+     * @return string
+     */
+    private function resourceNameHumanizer($resourceName)
```

Comment:
``resourceNameHumanizer`` seems like a service name. What do you think about ``humanizeResourceName(...)``?